tests/compile-fail/non_expressive_names.rs

```diff
@@ -0,0 +1,122 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+#![deny(clippy)]
+//~^ NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
+//~| NOTE: lint level defined here
```

Comment:
That’s weird, we never had to include that before. Must be because of the last upgrade of compiletest, but why is it needed only here?